python/tk_nuke_writenode/handler.py

```diff
@@ -2228,7 +2228,7 @@ def __on_script_save(self):
             self.__update_knob_value(
                 n,
                 "tk_write_node_settings",
-                six.ensure_text(base64.b64encode(knob_changes)),
+                sgutils.ensure_text(base64.b64encode(knob_changes)),
```

Comment:
Is there any way we can get rid of these functions?

Reply:
They're pretty much doing `encode()` and `decode()` under the hood with type safety checks. Not sure if it's worth the safety checks though.